style.css

```diff
@@ -69,10 +70,8 @@ a {
   top: 0;
   background-size: cover;
   background-repeat: no-repeat;
```

Comment:
- [ ] [OPTIONAL] I would suggest to use background-size: contain, this will make sure the background Image is displayed properly and will match the original figma as well :+1: